azurelinuxagent/common/protocol/__init__.py

```diff
@@ -15,7 +15,3 @@
 # Requires Python 2.6+ and Openssl 1.0+
 #
 
-from azurelinuxagent.common.protocol.util import get_protocol_util, \
-                                                 OVF_FILE_NAME, \
-                                                 TAG_FILE_NAME
-
```

Comment:
This import was creating a dependency on protocol.util to code that imported anything under protocol and was leading to a couple of circular dependencies. I remove the statement and updated the code to always import  get_procotol_util from protocol.util instead of protocol.